recording_server/src/media_context.ts

```
@@ -102,107 +186,213 @@ export class SoundContext extends MediaContext {
 
     public default() {
-        SoundContext.instance.play(`../silence.opus`, false)
+        // Play a default silence file
+        const silenceFile = IS_MACOS ? '../silence.wav' : '../silence.opus'
+        SoundContext.instance.play(silenceFile, false)
     }
 
     public play(pathname: string, loop: boolean) {
-        // ffmpeg -stream_loop -1 -re -i La_bataille_de_Farador.mp4 -f alsa -ac 2 -ar 44100 hw:Loopback,1
-        // ffmpeg -re -i cow_sound.mp3 -f alsa -acodec pcm_s16le "pulse:virtual_mic"
         let args: string[] = []
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'args' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)